test_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "frequency": 5,
+  "timeout": 1,
+  "uri": "http://example.org/api/v4/update",
+  "common_tags": {
```

Comment:
You might to add a test case  without common_tags.  It might initialize the `CommonTags` field to nil which can cause fun runtime exceptions if you are not checking for them.